Miscellaneous/scf-dictionary

```diff
@@ -0,0 +1,49 @@
+import pickle
+
+from wrap_pyscf import *
+
+scf_guess = {}
+
+filename = "scf_record"
+
+#with open(filename, 'rb') as outfile:
+    #scf_guess = pickle.load(outfile)
+
```

Comment:
There are interesting issues occurring. My computer can't finish running all of the "if" statements.  If I remove all "if" statements from this file except one (I used ethane), the scf calculations are executed and all that is contained in "scf_record" is an error saying that the file itself is not in 8 bit unicode (UTF-8). How can I fix that?

Reply:
And yes, I removed the wrap_pyscf api alterations and restarted the kernel.

Reply:
what line is raising the issue? what's the exact error.

Reply:
Never mind, I saw a problem where there actually wasn't one.